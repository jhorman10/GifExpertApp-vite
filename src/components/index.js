//archivo indice o de barril
export * from './AddCategory';
export * from './GifGrid';
export * from './GifGridItem';
export * from './Loader';
