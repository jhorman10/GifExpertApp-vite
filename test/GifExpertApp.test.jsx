import { render, screen } from '@testing-library/react';
import { GifExpertApp } from '../src/GifExpertApp';
describe('Pruebas en <GifExpertApp />', () => {
  test('Debe renderizar los Gifs haciendo match con el snapshot', () => {
    //TODO
    const { container } = render(<GifExpertApp />);
    expect(container).toMatchSnapshot();
    screen.debug();
  });
});
