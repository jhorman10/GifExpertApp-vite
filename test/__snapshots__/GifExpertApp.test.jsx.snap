// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifExpertApp /> Debe renderizar los Gifs haciendo match con el snapshot 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      name="category"
      placeholder="Buscar GIF"
      type="text"
      value=""
    />
  </form>
  <div
    class="main-container"
  >
    <h3>
      One Punch
    </h3>
    <div
      class="lds-facebook"
      title="loader"
    >
      <div />
      <div />
      <div />
    </div>
    <div
      class="card-grid"
    />
  </div>
</div>
`;
